src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -200,7 +200,6 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('pull_request_trigger_enabled', arg_group='Trigger', help="Indicates whether the source control pull request trigger is enabled. The trigger is disabled by default.", arg_type=get_three_state_flag())
         c.argument('schedule', arg_group='Trigger', help="Schedule for a timer trigger represented as a cron expression. An optional trigger name can be specified using `--schedule name:schedule` format. Multiples supported by passing --schedule multiple times.", action='append')
         c.argument('git_access_token', arg_group='Trigger', help="The access token used to access the source control provider.")
-        c.argument('branch', arg_group='Trigger', deprecate_info=c.deprecate(hide=True), help="The source control branch name. Please specify your source branch in the context parameter e.g. https://github.com/Azure-Samples/acr-build-helloworld-node.git#mybranch")
```

Comment:
what's behavior for user input branch already?

Reply:
user can set the branch of their git project where they want to read the file from.
But now we require the client to pass the branch with the context using the following format:
URI#branch:folder
For more formating details you can refer to https://docs.docker.com/engine/reference/commandline/build/#git-repositories